great-idea/index.html

```diff
@@ -15,9 +15,67 @@
 </head>
 
   <body>
-
-    <!-- Copy and paste your HTML from the first UI project here -->
-      
-    
+  <header>
+  <nav class="navigation">
+    <a href="services.html">Services</a>
+    <a href="">Product</a>
```

Comment:
```suggestion
    <a href="#">Product</a>
```
When you want to use an anchor tag without a link, put '#' in the href